app/assets/stylesheets/modules/_user_profiles.styl

```diff
@@ -216,3 +216,13 @@
 
 						p
 							font-size 10px
+
+html
+	scroll-behavior smooth
+	overflow scroll
+	scroll-padding-top 100px
+
+#userprofile_navbar
+	position sticky
+	top 50px
+	z-index 10
```

Comment:
What are these changes for?

Reply:
I've removed [`users_profile.js`](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/2f6fdea8e9c41698591754eb8c158f27955b63fc/app/assets/javascripts/utils/users_profile.js) and implemented the functionality using CSS instead.

Reply:
Whoops, I missed that. :+1: